tools/setup_helpers/cuda.py

```diff
@@ -20,10 +23,13 @@ def find_nvcc():
     WITH_CUDA = False
     CUDA_HOME = None
 else:
-    CUDA_HOME = os.getenv('CUDA_HOME', '/usr/local/cuda')
+    osname = platform.system()
+    if osname != 'Windows':
+        CUDA_HOME = os.getenv('CUDA_HOME', LINUX_HOME)
+    else:
+        CUDA_HOME = os.getenv('CUDA_PATH', WINDOWS_HOME).replace('\\', '/')
```

Comment:
what is this replaced used for? there's no `\\` in the string

Reply:
There're '\\' in the paths of the windows env variables.

Reply:
Oh right. I misread the code and missed that `WINDOWS_HOME` is only a default for `CUDA_PATH`